CHANGELOG.md

```diff
@@ -120,6 +120,8 @@ Changelog
   
 - `noise.estimate_assignment_probs()` now accepts a `QuantumComputer` instead of `QVMConnection`.
   
+- `RX` and `RZ` gate fidelities are passed to Quilc as the _simultaneous_, rather than independent, random benchmark for those respective operations.
```

Comment:
Agreed this should be committed as a "fix" if it is fixing something from 2.x

Reply:
Originally, we passed [f1QRB](https://github.com/rigetti/pyquil/blob/master/pyquil/device/_specs.py#L74) random benchmark scores to quilc.

When I was discussing this with Mark Hodson, I asked if we should use the independent or simultaneous RB score. He didn't have a strong opinion, but if he had to lean one way or another, it would be toward the simultaneous score which should _generally_ be more representative of operations run in the context of a circuit, rather than a single isolated operation.

Reply:
The original implementation isn't a bug (perhaps simultaneous scores weren't available at the time), which is why I called this an update.

The _bug_ this PR addresses is that the original changes I made to pyQuil v3 did not pass _any_ fidelity value from the server specs to the compiler, but instead used hard coded default values.